### PR TITLE
realtikvtest: dedicated package for real tikv tests

### PR DESCRIPTION
**Pull request by tisonkun**
Issue Number: ref #28328

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

**Comment by tisonkun** [comment]
@hawkingrei this new workflow is very unstable due to the lack of resource on GitHub Actions. It'd be better to involve related developers for a better CI resource.

**Comment by xhebox** [comment]
> @hawkingrei this new workflow is very unstable due to the lack of resource on GitHub Actions. It'd be better to involve related developers for a better CI resource.

Agree.

File: .github/workflows/integration-test-with-real-tikv.yml [hunk 1]
```diff
@@ -33,3 +33,25 @@ jobs:
           export log_level=error
           make failpoint-enable
           go test ./tests/pessimistictest -v -with-real-tikv -timeout 20m
+  realtikv-test:
+    runs-on: ubuntu-latest
```
Comment (tisonkun) [inline .github/workflows/integration-test-with-real-tikv.yml:37]:
```suggestion
    runs-on: [self-hosted, X64]
```

**Comment by tisonkun** [comment]
@hawkingrei any news about test resources?

**Comment by hawkingrei** [comment]
> @hawkingrei any news about test resources?

Wait for EE team after the holiday.

**Comment by tisonkun** [comment]
@hawkingrei @xhebox code updated. Waiting for https://github.com/PingCAP-QE/ci/pull/953.

**Comment by tisonkun** [comment]
/hold

While migrate in place, figure out how to deal with test-infra later.

**Pull request closed without merging**